.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage/*
 notes.md
 tests/fixtures/cassette*
+composer.lock
```

Comment:
i have to respectfully but strongly disagree ;-)

being a library, the library lock file is not used by the users of php-vcr. the only thing tracking the lock file can do here is hiding problems. if this was an application, the assumption that the lock file helps would be correct. but in a reusable library, any user might at any point update dependencies. if composer.json does not limit dependencies to working versions, things will fail for them.